From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Thu, 31 Oct 2024 16:35:11 -0500
Subject: [PATCH] solus: Set default PID

Technically this isn't used because we're not shipping a default configuration file, but if the user overrides that let's ensure that we're using the right path

---
 src/server.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/server.h b/src/server.h
index 36668ae31..c4bb54c6e 100644
--- a/src/server.h
+++ b/src/server.h
@@ -130,7 +130,7 @@ struct hdr_histogram;
 #define RDB_EOF_MARK_SIZE 40
 #define CONFIG_REPL_BACKLOG_MIN_SIZE (1024 * 16) /* 16k */
 #define CONFIG_BGSAVE_RETRY_DELAY 5              /* Wait a few secs before trying again. */
-#define CONFIG_DEFAULT_PID_FILE "/var/run/valkey.pid"
+#define CONFIG_DEFAULT_PID_FILE "/run/valkey/valkey.pid"
 #define CONFIG_DEFAULT_BINDADDR_COUNT 2
 #define CONFIG_DEFAULT_BINDADDR {"*", "-::*"}
 #define NET_HOST_STR_LEN 256                          /* Longest valid hostname */
